### DISCUSSION TITLE
hide draggable space on unmaximized window

### DISCUSSION THREAD

**Discussion by shalva97** [minbrowser/min#1754] [Ideas]
I have got a new monitor, which is Ultra wide  3440x1440

the problem I have got now is that I place 2 windows side by side so none of them are maximized. Draggable space about Min windows is always visible :/

![image](https://user-images.githubusercontent.com/22417494/134949412-702cf12e-1905-4332-af6d-38deca907c7b.png)

**Comment by PalmerAL** [comment]
Hm, we'd need a drag space somewhere, or you couldn't move the window into place on the left side...

**Reply by shalva97** [reply]
yeah, thats true. But how do other people use wide monitors.? hmmm....
